androidtv/constants.py

```diff
@@ -323,15 +346,20 @@
         APP_PLAY_VIDEOS: 'Play Videos',
         APP_PLEX: 'Plex',
         APP_PRIME_VIDEO: 'Prime Video',
+        APP_PRIME_VIDEO_2: 'Prime Video',
```

Comment:
The friendly names need to be unique. Same goes for YouTube.

Reply:
@JeffLIrion How would you like to handle when an app has two different identifiers (i.e. YouTube)? What should it be renamed as?

Reply:
I'll leave that up to you. For YouTube, maybe "YouTube (Fire TV)". I'm not sure what the difference is between these two Amazon Prime Video apps.

Reply:
I noticed that a few of them mention `eu`, but I'm in North America so I'm wondering if it's possibly just a regional difference